www/friendly.html.pm

```diff
@@ -1,6 +1,10 @@
 #lang pollen
 
-◊(require (only-in "index.html.pm" top))
+◊(require (only-in "index.html.pm" top)
+          racket/string)
+
+◊(define (email user . host)
+   (format "~a at ~a" user (string-join host " dot ")))
 
```

Comment:
I think that obfuscating the email addresses in this is no barrier to the spammers, only a (mild) barrier to the legit folks who want to contact the moderation team. If it were me, I'd put in mailto: links.

Reply:
From the discussion above, though, at least two of the owners of the email addresses wanted light obfuscation.

Reply:
I saw one person saying both they'd do it that way on their own and also questioning whether it was a good idea to do that way; I didn't see a second person. (I must have missed a comment.) Anyway, I'm not one of the people whose emails are here, tho. I just posted because it seems like making it easy to contact these folks is worth doing.

Reply:
I’m not bothered. My email is easily found.

Reply:
After checking with Sam, it sounds like everyone is happy with mailto links after all, so I pushed that change.

Reply:
Thanks!

Reply:
Works for me!